package.json

```diff
@@ -81,7 +81,8 @@
                         "bat": "",
                         "shellscript": "bash",
                         "fsharp": "fsi",
-                        "csharp": "scriptcs"
+                        "csharp": "scriptcs",
+                        "vbscript": "cscript //Nologo"
```

Comment:
Cool! It's fine with me if you want to implement the config.  I'm not well versed in TypeScript yet, but I'm trying. :smile: